codecept.conf.js

```diff
@@ -0,0 +1,24 @@
+const { setHeadlessWhen, setCommonPlugins } = require('@codeceptjs/configure');
+// turn on headless mode when running with HEADLESS=true environment variable
+// export HEADLESS=true && npx codeceptjs run
+setHeadlessWhen(process.env.HEADLESS);
+
+// enable all common plugins https://github.com/codeceptjs/configure#setcommonplugins
+setCommonPlugins();
+
+/** @type {CodeceptJS.MainConfig} */
+exports.config = {
+  tests: './*_test.js',
+  output: './output',
+  helpers: {
+    Playwright: {
+      browser: 'chromium',
+      url: 'http://localhost:8080',
```

Comment:
```suggestion
      url: 'http://localhost:3000',
```
next 포트에 맞추어야 제대로 동작할 것 같습니다